releasenotes/notes/additional-endpoints-config-from-env-vars-a5db51083cbd310f.yaml

```diff
@@ -0,0 +1,6 @@
+enhancements:
+  - |
+    Add the ability to configure additional endpoints for
+    both the agent and the log agent with environment
+    variables, namely DD_ADDITIONAL_ENDPOINTS and
+    DD_LOGS_CONFIG_ADDITIONAL_ENDPOINTS.
```

Comment:
let's remove the release note: technically `additional endpoints` is a hidden feature that we don't want to advertise since it has billing implications. Users should only use it after they've contacted support.

Reply:
It makes sense, release note dropped.